etc/templates/error/502.html

```diff
@@ -0,0 +1,81 @@
+        </div>
+
+        <div id="login" class="standard_form inlined">
+	        <p>Site is currently down for maintenance. Please try again later.</p>
+
+        </div>
+
+        <div id="login-footer">
+	        <p>
+		        &copy; 2007-2014 University of Dundee &amp; Open Microscopy Environment<br>
```

Comment:
Sorry what do you mean? As a `php` or `script` line in the HTML itself that would return the current year or as a automated way to update the source code?